src/options.js

```diff
@@ -365,7 +365,8 @@ exports.bool = {
      *
      * For more info about strict mode see the `strict` option.
      *
-     * @deprecated Use `strict: "global"`.
+     * @deprecated Use `strict: global` is `strict` was `true`,
+     *             or `strict: ignore` is it was `false`.
```

Comment:
It should be `Use 'strict: ignore' to allow a global "use strict" directive or 'strict: global' to require it.`